### ISSUE TITLE
Boot shows ``Err(0xE) <- RT.GV wake-failure``

### ISSUE THREAD

**Issue by jclsn** [BillDH2k/Hackintosh-DELL-T5810-OpenCore#9]
I can't boot Sonoma or Ventura. Getting the same error as described here https://www.reddit.com/r/hackintosh/comments/1bvd5jn/rtgv_wakefailure_preventing_me_from_running/

Has anyone else seen this?

**Comment by BillDH2k** [comment]
T5810 hardware? T5810 is based on X99, not X299. Also, your error may be related to FV-919.

**Comment by jclsn** [comment]
Yes, it is exactly a T5810. The only difference may be the Radeon 6600 XT, but I have activated the required boot flag for the GPU. 

What is FV-919?

**Comment by jclsn** [comment]
The configuration from Olarila shows the same problem. Maybe it is one of the PCI devices. I have a WLAN/BT card in there. Or can USB devices also prevent boot?

**Comment by BillDH2k** [comment]
Fenvi-T919 Wifi/BT Card? Try without it.

**Comment by jclsn** [comment]
I don’t own this card. I have an Intel card installed. Tried taking it out, but it didn’t make a difference.
